### PR TITLE
[refactor] Adjust the code style in `pkg/cmd/volume/*`

### PR DESCRIPTION
**Pull request by Zheaoli**
Signed-off-by: Zheao.Li <me@manjusaka.me>

### CODE REVIEW

File: cmd/nerdctl/volume_prune.go [hunk 1]
```diff
@@ -35,17 +35,25 @@ func newVolumePruneCommand() *cobra.Command {
...
-	}, cmd.InOrStdin(), cmd.OutOrStdout())
+	}, nil
+}
+
+func volumePruneAction(cmd *cobra.Command, _ []string) error {
+	options, err := processVolumePruneCommandOptions(cmd)
+	if err != nil {
+		return err
+	}
+	return volume.Prune(cmd.Context(), options, cmd.InOrStdin(), cmd.OutOrStdout())
```
Comment (djdongjin) [inline cmd/nerdctl/volume_prune.go:58]: curious why only `Prune` and `Rm` requires `ctx` (`cmd.Context()`) and other commands don't 🤔

File: cmd/nerdctl/volume_rm.go [hunk 2]
```diff
@@ -38,15 +38,27 @@ func newVolumeRmCommand() *cobra.Command {
...
+		Force:    force,
+	}, nil
+}
+
+func volumeRmAction(cmd *cobra.Command, args []string) error {
+	options, err := processVolumeRmCommandOptions(cmd)
+	if err != nil {
+		return err
+	}
+	return volume.Rm(cmd.Context(), args, options, cmd.OutOrStdout())
```
Comment (djdongjin) [inline cmd/nerdctl/volume_rm.go:61]: `Rm` -> `Remove`

File: pkg/api/types/volume_types.go [hunk 3]
```diff
@@ -54,6 +50,6 @@ type VolumePruneCommandOptions struct {
 
 type VolumeRmCommandOptions struct {
```
Comment (djdongjin) [inline pkg/api/types/volume_types.go:51]: `Rm` -> `Remove`

File: cmd/nerdctl/volume_prune.go (hunk 1 above)
Comment (Zheaoli) [reply cmd/nerdctl/volume_prune.go:58]: We need to check if there is any volumes which is used by running container

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**